pkg/processor/cascadingfilterprocessor/cascade_test.go

```diff
@@ -33,6 +33,8 @@ import (
 var testValue = 10 * time.Millisecond
 var probabilisticFilteringRate = int32(10)
 var healthCheckPattern = "health"
+var statusCode = "ERROR"
```

Comment:
I would change it to `ptrace.StatusCodeError.String()` and removed `mapStatusCode`

Reply:
done!